packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-settings/pipeline-menus.tsx

```diff
@@ -119,26 +45,30 @@ export const SaveMenuComponent: React.FunctionComponent<SaveMenuProps> = ({
         return onCreateView();
     }
   };
+  const actions = useMemo(
+    () =>
+      [
+        ...saveMenuActions,
+        isCreateViewAvailable && {
+          action: 'createView',
+          label: 'Create view',
+        },
+      ].filter(Boolean) as MenuAction<SaveMenuActions>[],
```

Comment:
Completely optional suggestion, but if you want to avoid the TS cast here:

```suggestion
      saveMenuActions.concat(
        isCreateViewAvailable ? [{
          action: 'createView',
          label: 'Create view',
        }] : []),
```